docs/reference/contributing/target/rtos.md

```diff
@@ -1,75 +1,31 @@
 ### RTOS
 
-CMSIS/RTX code is imported from [the original CMSIS repository](https://github.com/ARM-software/CMSIS_5/).
+CMSIS/RTX code is imported from [the original CMSIS repository](https://github.com/ARM-software/CMSIS_5/) with the help of a Python script in the `mbed-os` repository `mbed-os\tools\importer\importer.py`.
 
 #### Memory considerations
 
-Please note that Arm Mbed OS doesn't use any of the RTX memory models, which are based on static carveouts (memory pools). This approach is not ideal for generic systems, such as Mbed OS, because calculating required numbers of RTOS objects is impossible. To avoid declaring arbitrary large buffers carved out at compile time, limiting the amount of available memory, Mbed OS shifts the responsibility of supplying the backing memory to CMSIS-RTOS2 users.
+Please note that Arm Mbed OS doesn't use any of the RTX memory models, which are based on static carveouts (memory pools). This approach is not ideal for platform operating systems, such as Mbed OS, because calculating required numbers of RTOS objects is impossible. To avoid declaring arbitrary large buffers carved out at compile time, limiting the amount of available memory, Mbed OS shifts the responsibility of supplying the backing memory to CMSIS-RTOS2 users.
```

Comment:
I'm not sure what the `platform operating systems` are to be fair.

Reply:
@bulislaw - That change was review comment by @sg- https://github.com/ARMmbed/Handbook/pull/340/files/d827afd63fab6edc4021eec3ddce567217a229d6#diff-d3a387b212d57c83382162e66ce314a0R3

Reply:
Fair enough.